src/Module/Settings/UserExport.php

```diff
@@ -162,9 +169,9 @@ private static function exportContactsAsCSV()
 		// write the table header (like Mastodon)
 		echo "Account address, Show boosts\n";
 		// get all the contacts
-		$contacts = DBA::select('contact', ['addr'], ['uid' => $_SESSION['uid'], 'self' => false, 'rel' => [1,3], 'deleted' => false]);
+		$contacts = DBA::select('contact', ['addr', 'url'], ['uid' => $_SESSION['uid'], 'self' => false, 'rel' => [1,3], 'deleted' => false]);
 		while ($contact = DBA::fetch($contacts)) {
-			echo $contact['addr'] . ", true\n";
+			echo ($contact['addr'] ? $contact['addr'] : $contact['url']) . ", true\n";
```

Comment:
Small readability improvement
```suggestion
			echo ($contact['addr'] ?: $contact['url']) . ", true\n"
```